bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/thing/screen.xml

```diff
@@ -9,6 +9,7 @@
 			<bridge-type-ref id="bridge"/>
 		</supported-bridge-type-refs>
 		<label>Somfy Screen</label>
+		<semantic-equipment-tag>WindowCovering</semantic-equipment-tag>
```

Comment:
Why is an Exterior Screen tagged `Blinds` and these `WindowCoverings` ?
bundles/org.openhab.binding.somfytahoma/src/main/resources/OH-INF/thing/exteriorscreen.xml

Better be consistent, not sure which one to fix.

Reply:
Aligned.